handlers/deltaValidationHandler.go

```diff
@@ -0,0 +1,50 @@
+package handlers
+
+import (
+	"fmt"
+	"github.com/companieshouse/chs-delta-api/config"
+	"github.com/companieshouse/chs-delta-api/helpers"
+	"github.com/companieshouse/chs-delta-api/validation"
+	"github.com/companieshouse/chs.go/log"
+	"net/http"
+)
+
+// DeltaValidationHandler offers a handler by which to publish an office-delta onto the officer-delta kafka topic.
+type DeltaValidationHandler struct {
+	h    helpers.Helper
+	chv  validation.CHValidator
+	cfg  *config.Config
+}
+
+// NewDeltaValidationHandler returns an DeltaValidatorHandler.
+func NewDeltaValidationHandler(h helpers.Helper, chv validation.CHValidator, cfg *config.Config) *DeltaValidationHandler {
+	return &DeltaValidationHandler{h: h, chv: chv, cfg: cfg}
+}
+
+// ServeHTTP accepts an incoming OfficerDelta request via a POST method, validates it
+// and then passes it to a Kafka service for further processing along with an officers-delta topic. If errors are
```

Comment:
This comment above the ServeHTTP method will need altering to remove mentions of sending to Kafka. Probably best to make it clear this is validation only

Reply:
Yeah, spotted that and updated 👍